package.json

```diff
@@ -2,6 +2,7 @@
   "name": "ember-frost-button",
   "version": "0.1.0",
   "description": "Simple Frost Button",
+  "frost_guide_directory": "ui-components/button-controls/button",
```

Comment:
Is there a reason this has to be `snake_case`? Can we update the guide to look for `frostGuideDirectory` instead? It would be much more consistent with other `package.json` keys.